internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -371,70 +308,28 @@ func convertTaskDefOverrideRules(inRules []manifest.OverrideRule) []override.Rul
 	return res
 }
 
-func isValidTaskDefOverridePath(path string) bool {
-	for _, s := range invalidTaskDefOverridePathRegexp {
-		re := regexp.MustCompile(fmt.Sprintf(`^%s$`, s))
-		if re.MatchString(path) {
-			return false
-		}
-	}
-	return true
-}
-
 // convertStorageOpts converts a manifest Storage field into template data structures which can be used
 // to execute CFN templates
 func convertStorageOpts(wlName *string, in manifest.Storage) (*template.StorageOpts, error) {
```

Comment:
Seems like the function signature can be changed now!
```suggestion
func convertStorageOpts(wlName *string, in manifest.Storage) *template.StorageOpts {
```